RecoBTag/Combined/plugins/BTagProbabilityToDiscriminator.cc

```diff
@@ -161,8 +161,54 @@ void
 BTagProbabilityToDiscriminator::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
   //The following says we do not know what parameters are allowed so do no validation
   // Please change this to state exactly what you do use, even if it is no parameters
-  edm::ParameterSetDescription desc;
-  desc.setUnknown();
+	edm::ParameterSetDescription desc;
```

Comment:
Please indent coherently the code in the whole method: as it is now it is rather difficult to read it